src/index.html

```diff
@@ -6,9 +6,54 @@
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap">
     <link rel="stylesheet" href="./style.css">
   </head>
+
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#home" class="logo">
+        <img
+        src="images/Logo.png"
+        alt="Moyo logo"
+        class="logo__img"
+        >
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a href="#Apple" class="nav__link is-active">Apple</a>
+          </li>
+          <li class="nav__item">
```

Comment:
it's a good practice to add an empty line between _**multiline**_ siblings for better readability

```suggestion
          </li>
          
          <li class="nav__item">
```